sdk-extensions/tracing-incubator/src/jmh/java/io/opentelemetry/sdk/extension/incubator/trace/BatchSpanProcessorMetrics.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.extension.incubator.trace;
+
+import io.opentelemetry.sdk.metrics.data.LongPointData;
+import io.opentelemetry.sdk.metrics.data.MetricData;
+import java.util.Collection;
+import java.util.OptionalLong;
+
+public class BatchSpanProcessorMetrics {
```

Comment:
does this class need to be public? I don't think we'd want to expose it as a part of the API, even in the incubator.

Reply:
never mind...it's in the jmh package. 🤦

Reply:
It would be really cool if we had a separate module for the jmh benchmarks, so we could share all of this code and not make copies of it. Can you create an issue to track that work?

Reply:
I've created an issue https://github.com/open-telemetry/opentelemetry-java/issues/3083